src/formatters/backgroundFormatter.ts

```diff
@@ -0,0 +1,32 @@
+import { Background } from "gherkin-ast";
+import { FormatOptions } from "../index";
+import { config, indent, lines } from "../utils";
+import { format as formatStep } from "./stepFormatter";
+import { getDebugger } from '../debug';
+
+const debug = getDebugger("backgroundFormatter");
+
+export function format(background: Background, options: Partial<FormatOptions>): string {
+    debug("format(background: %s, options: %o)", background.constructor.name, options);
```

Comment:
For all files, where you use such debug statement, please set a `?.` for the constructor, to ensure it won't fail if a falsy value is passed.